marimo/_runtime/runtime.py

```
@@ -2090,6 +2091,7 @@ def launch_kernel(
     control_queue: QueueType[ControlRequest],
     set_ui_element_queue: QueueType[SetUIElementValueRequest],
     completion_queue: QueueType[CodeCompletionRequest],
+    mcp_evaluation_queue: QueueType[MCPServerEvaluationRequest],
```

Ruff (0.8.2)

Unused function argument: `mcp_evaluation_queue`

(ARG001)